test/plugin.js

```diff
@@ -15,7 +15,7 @@ describe('Plugin', function () {
 describe('Plugin', function () {
   let nightmare;
 
-  this.timeout(3000);
+  this.timeout(5000);
```

Comment:
Can we play it safe and increase this to a large enough value that should not timeout in the future, either? Maybe something like 15 seconds should be safe enough?

```suggestion
  this.timeout(15000);
```